src/bridge/ex_data_util/ex_data_util_test.cc

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "src/bridge/ex_data_util/ex_data_util.h"
+#include "src/bridge/memory_util/openssl_structs.h"
+#include "src/testing_util/mock_client.h"
+#include "src/testing_util/mock_rsa_key.h"
+#include "src/testing_util/test_matchers.h"
+
+namespace kmsengine {
+namespace bridge {
+namespace {
+
+using ::kmsengine::testing_util::IsOk;
+using ::kmsengine::testing_util::MockRsaKey;
+using ::testing::Not;
+
+TEST(ExDataUtilTest, RsaKeyRoundtrip) {
+  ASSERT_THAT(InitExternalIndicies(), IsOk());
+
+  auto rsa = MakeRsa();
```

Comment:
Similar to the comment above (and for the whole file), prefer to spell out the return type explicitly unless it's really obvious: https://google.github.io/styleguide/cppguide.html#Type_deduction